evap/staff/tests/test_views.py

```diff
@@ -899,6 +901,28 @@ def test_success_with_data(self):
 
         self.assertFalse(Semester.objects.filter(pk=self.instance.pk).exists())
 
+    @override_settings(REWARD_POINTS=[(1, 3)])
+    def test_does_not_grant_redemption_points(self):
+        student = baker.make(UserProfile, email="student@institution.example.com")
+        evaluation = baker.make(Evaluation, participants=[student], state=Evaluation.State.PUBLISHED, course__semester=self.instance)
+        SemesterActivation.objects.update_or_create(semester=self.instance, defaults={"is_active": True})
```

Comment:
Why update _or_ create? Shouldn't we be in control of whether or not one exists already?

Reply:
My thought here was that the test doesn't care whether `Semester.create` / `save` / the login in `setUpTestData` will create a semester activation, and if it does, what its value is. All we want is the semester to be active. This line generically gets us to that state, independently of what state we were in before.